var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -38,10 +40,14 @@ class Scotch(Package):
     version('6.0.0', 'c50d6187462ba801f9a82133ee666e8e')
     version('5.1.10b', 'f587201d6cf5cf63527182fbfba70753')
 
-    variant('mpi', default=False, description='Activate the compilation of PT-Scotch')
-    variant('compression', default=True, description='Activate the posibility to use compressed files')
-    variant('esmumps', default=False, description='Activate the compilation of the lib esmumps needed by mumps')
-    variant('shared', default=True, description='Build shared libraries')
+    variant('mpi', default=False,
```

Comment:
@davydden, @adamjstewart: I would really like for functions like `variant` and `version` to be exempted from the line limit as well, and it would be fairly simple to add.  Would this change be okay with you @tgamblin?